/taser/**:
  ros__parameters:
    dt: 0.01
    initial_pose:
      x: -2.0
      y: 0.0
      z: 0.65
      qw: 1.0
      qx: 0.0
      qy: 0.0
      qz: 0.0

    locomotion:
      wheel_base: 0.6 # Obtained using transform from left wheel to right wheel
      wheel_radius: 0.15

    manipulation:
      controller_kp: 5.0

    navigation:
      workspace:
        x_min: -6.0
        x_max: 6.0
        y_min: -6.0
        y_max: 6.0
      v_max: 2.0
      w_max: 1.0
      num_rrt_samples: 200
      mpc_horizon: 30
      polygons:
        _polygon_count: 3
        polygon_0:
          position: [0.0, 0.0, 0.5]
          size: [0.5, 4.0, 1.0]
        polygon_1:
          position: [1.75, -2.0, 0.5]
          size: [4.0, 0.5, 1.0]
        polygon_2:
          position: [-1.75, 2.0, 0.5]
          size: [4.0, 0.5, 1.0]
